samples/docker-domain/container-scripts/startWLSDomain.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
+#Copyright (c) 2014-2018 Oracle and/or its affiliates. All rights reserved.
+#
+#Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+#
+# Start the Domain.
+
+# determine the domain name. there is only one domain directory.
+export DOMAIN_NAME=`ls /u01/oracle/user_projects/domains | head -1`
+
+#Define DOMAIN_HOME
+export DOMAIN_HOME=/u01/oracle/user_projects/domains/$DOMAIN_NAME
+
+# Start Admin Server and tail the logs
+${DOMAIN_HOME}/startWebLogic.sh
+touch ${DOMAIN_HOME}/servers/AdminServer/logs/AdminServer.log
```

Comment:
We should discuss this as for K8s we're going to want the logs relocated outside the domain home to a file-system location where a persistent volume is mounted.

Reply:
This is planned to have a dynamic cluster, still working on that. I think the log directory can be moved to persistent volume.